keycloak-event-emitter-tests/src/test/java/io/cloudtrust/keycloak/AbstractTest.java

```diff
@@ -8,22 +9,39 @@
 import io.undertow.util.StatusCodes;
 import org.apache.commons.io.IOUtils;
 import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.test.api.ArquillianResource;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.ClassRule;
+import org.junit.*;
```

Comment:
This is not a best practice... I think Alistair told me there was a way to disable this automatic "cleaning" in import organization in IntelliJ (but I don't remember how)

Reply:
Whenever needed: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html
But you need to set a big value in the counts.... Not very nice.